deploy/localhost.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/bash
+#
+#    Deploy cert to localhost similar to certbot behavior
+#
+#    export DEPLOY_LOCALHOST_PATH="/path/to/certs"
```

Comment:
the variable name should be something like :  `DEPLOY_LOCALHOST_ROOT_PATH`

Reply:
I changed the bash line and renamed DEPLOY_LOCALHOST_ROOT_PATH. What else do i need to do?